x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group4/builtin_alert_types/long_running/rule.ts

```diff
@@ -110,11 +110,13 @@ export default function ruleTests({ getService }: FtrProviderContext) {
         });
       });
 
-      const { status, body: rule } = await supertest.get(
-        `${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule/${ruleId}`
-      );
-      expect(status).to.eql(200);
-      expect(rule.execution_status.status).to.eql('active');
+      await retry.try(async () => {
+        const { status, body: rule } = await supertest.get(
+          `${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule/${ruleId}`
+        );
+        expect(status).to.eql(200);
+        expect(rule.execution_status.status).to.eql('active');
+      });
```

Comment:
Sometimes the event log documents are persisted before the rule finishes running and update its own status, causing this code to expect `active` while the rule is not updated yet.